src/polar/Radar.tsx

```diff
@@ -251,16 +256,69 @@ function Dots({ points, props }: { points: ReadonlyArray<RadarPoint>; props: Pro
   return <Layer className="recharts-radar-dots">{dots}</Layer>;
 }
 
+function RadarLabelListProvider({
+  showLabels,
+  points,
+  children,
+}: {
+  showLabels: boolean;
+  points: ReadonlyArray<RadarPoint>;
+  children: ReactNode;
+}) {
+  /*
+   * Radar provides a Cartesian label list context. Do we want to also provide a polar label list context?
+   * That way, users can choose to use polar positions for the Radar labels.
+   */
+  // const labelListEntries: ReadonlyArray<PolarLabelListEntry> = points.map(
```

Comment:
delete?

Reply:
I think that Radar being a polar chart should also support polar label positions, so I propose to pass both viewboxes and then use one or the other depending on the position? I didn't do this here because the PR is already large enough.

Reply:
Sounds good